### PR TITLE
add troubleshooting section to readme

### PR DESCRIPTION
**Pull request by greyepoxy**
# Description

Add troubleshooting section to readme

Fixes # n/a

## Type of Change

Compatible

Documentation only

# Testing

n/a

# Checklist

- [x] Have you [signed the CLA](contributing.md#sign-the-cla)?
- ~~[ ] Has the unreleased [Changelog](changelog.md#unreleased) section been updated with information about this change?~~
- [x] Has the [readme](readme.md) been updated to reflect this change?
- ~~[ ] Are there tests validating this change?~~

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -312,6 +312,24 @@ algorithm](https://nodejs.org/api/modules.html#modules_require_resolve_request_o
...
 
+## Troubleshooting
+
+### Npm-pack-here runs but webpack (or other bundler) doesn’t detect any changes
+
+As of Webpack v5 the default `node_modules` caching settings will likely cause issues when attempting
+to do local testing with `npm-pack-here`. The problem is the
+[`snapshot.managedPaths`](https://webpack.js.org/configuration/other-options/#managedpaths) default
+value. It appears like webpack’s caching does not detect any changes to `file:` dependencies as different.
+To resolve this, remove the `node_modules` default value`snapshot.managedPaths = []`.
```
Comment (emclain) [inline README.md:323]: This is confusing.  Are you saying add `snapshot.managedPaths = []` to your config?  Or remove this line from your config?

Comment (greyepoxy) [reply README.md:323]: add, if its not set to empty then it picks up the default which is `snapshot.managedPaths = [path.resolve(__dirname, '../node_modules')]`

Comment (emclain) [reply README.md:323]: Can you say something like 

To resolve this, add `snapshot.managedPaths = []` to your configuration to override the default value (`node_modules`).

Comment (greyepoxy) [reply README.md]: what do you think of the changes?

Comment (emclain) [reply README.md]: lgtm!

**Review by emclain — APPROVED** [review]

**Pull request merged**